x-pack/plugins/security_solution/server/endpoint/routes/metadata/handlers.ts

```diff
@@ -167,52 +181,76 @@ export const getMetadataRequestHandler = function (
 export async function getHostData(
   metadataRequestContext: MetadataRequestContext,
   id: string,
-  queryStrategyVersion?: MetadataQueryStrategyVersions
+  queryStrategyVersion?: MetadataQueryStrategyVersions,
+  includeAgentData?: boolean
 ): Promise<HostInfo | undefined> {
+  if (!metadataRequestContext.savedObjectsClient) {
+    return undefined;
```

Comment:
This should probably throw an error, since a context dependency was not provided